app/src/main/java/io/appium/uiautomator2/model/UiObject2Element.java

```diff
@@ -60,6 +60,7 @@ public String getText() throws UiObjectNotFoundException {
         if (isToastElement(nodeInfo)) {
             return nodeInfo.getText().toString();
         }
+        nodeInfo = AccessibilityNodeInfoGetter.fromUiObject(element);
```

Comment:
what is the point of doing this?

Reply:
Because [getAccessibilityNodeInfo()](https://android.googlesource.com/platform/frameworks/uiautomator/+/android-support-test/src/main/java/android/support/test/uiautomator/UiObject2.java#619) refresh element's accessiblity info.
Without refreshing it returns old cached value of [getRangeInfo()](https://github.com/appium/appium-uiautomator2-server/blob/master/app/src/main/java/io/appium/uiautomator2/model/UiObject2Element.java#L64)

Reply:
then it makes sense to do this inside `ff` block

Reply:
laso add a comment why it is needed